.github/workflows/builds.yml

```diff
@@ -69,12 +69,27 @@ jobs:
         PLATFORM:
           - NAME: "CUDA"
             CONTAINER: "ghcr.io/acts-project/ubuntu1804_cuda:v11"
+            OPTIONS:
           - NAME: "CUDA"
             CONTAINER: "ghcr.io/acts-project/ubuntu2004_cuda:v13"
+            OPTIONS:
           - NAME: "HIP"
             CONTAINER: "ghcr.io/acts-project/ubuntu1804_rocm:v11"
+            OPTIONS:
           - NAME: "SYCL"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004_oneapi:v12"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2004_oneapi:v20"
+            OPTIONS:
+          - NAME: "SYCL"
+            CONTAINER: "ghcr.io/acts-project/ubuntu1804_cuda_oneapi:v20"
+            OPTIONS: -DVECMEM_BUILD_CUDA_LIBRARY=FALSE
+        include:
+          - BUILD:
+              TYPE: "Release"
+              MSG_LVL: 0
+            PLATFORM:
+              NAME: "SYCL"
+              CONTAINER: "ghcr.io/acts-project/ubuntu1804_rocm_oneapi:v20"
+              OPTIONS: -DVECMEM_BUILD_HIP_LIBRARY=FALSE
```

Comment:
Maybe I don't get the structure but the upper job does not build the CUDA lib but uses the CUDA oneAPI image, the lower one uses the rocm oneAPI image but goes not build the HIP library. 

I'm sure this makes sense, but why?

Reply:
For the "container builds" we only run unit tests for the SYCL builds. Because the CUDA and HIP tests both need NVIDIA and AMD GPUs to be available to run correctly.

By default these rocm_oneapi and cuda_oneapi packages make the project build both HIP/SYCL and CUDA/SYCL libraries/tests. At which point the CI would try to run the HIP and CUDA unit tests in them.

So it's not that the build for these would be harmful to enable. It's that I didn't want to try to specify that everything should be built in these images, but only the non-CUDA / non-HIP tests should be run.

Reply:
Ok I see.